lib/action_cable/channel/naming.rb

```diff
@@ -0,0 +1,23 @@
+module ActionCable
+  module Channel
+    module Naming
+      extend ActiveSupport::Concern
+
+      class_methods do
+        # Returns the last part of the channels's name, underscored, without the ending
+        # <tt>Channel</tt>. For instance, ChatChannel returns <tt>chat</tt>.
+        #
+        # ==== Returns
+        # * <tt>string</tt>
```

Comment:
@kaspath - You are right - `the last part` should not be there, and has been removed.